### ISSUE TITLE
Parse VFI as French

### ISSUE THREAD

**Issue by Billos** [Sonarr/Sonarr#6927]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

When parsing a media title, VFI is detected as English

### Expected Behavior

Should be detected as French

### Steps To Reproduce

Custom Format => Parsing => Testing the following string for instance : "Malcolm In The Middle S01 Multi VFI VO 1080p WEB x265 HEVC AAC 5.1-Papaya"

Language is detected as English

### Environment

```markdown
- OS: Debian 11
- Sonarr: 4.0.5.1710
- Docker Install: yes
- Using Reverse Proxy: yes 
- Browser: Firefox
- Database: Sqlite 3.45.3
```

### What branch are you running?

Main

### Trace Logs?

Does not generate log

### Anything else?

![image](https://github.com/Sonarr/Sonarr/assets/5809662/117ef2ef-596d-4b81-afd7-809fce10a089)
![image](https://github.com/Sonarr/Sonarr/assets/5809662/fc6f7b04-475c-4b8f-858e-293fc28f2249)

**Comment by thezoggy** [comment]
> Sonarr doesn't what `VFI` means, it sees no languages in that and uses the default language, which is English.
> 
> What does `VFI` stand for in this case?

for reference: https://github.com/Radarr/Radarr/commit/5f396a53c3d55cb95ea7ae93622555748fb02298

**Comment by Billos** [comment]
> Your example also has `VO` in it, which stands for `version originale`... So it's a multi-language release with original (English) and dubbed French audio tracks?

Yes, the media includes English and French audio
